### PR TITLE
docs: update global styles/scripts/lib wiki

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

File: docs/documentation/stories/global-styles.md [hunk 1]
```diff
@@ -6,36 +6,41 @@ The `styles.css` file allows users to add global styles and supports
...
-  "more-styles.css",
-],
+"architect": {
+  "build": {
+    "builder": "@angular-devkit/build-angular:browser",
+    "options": {
+      "styles": [
+        "src/styles.css",
+        "src/more-styles.css",
+      ],
```
Comment (Brocco) [inline docs/documentation/stories/global-styles.md]: please close curly braces

Comment (filipesilva) [reply docs/documentation/stories/global-styles.md]: Added `...` to indicate more options are there.

File: docs/documentation/stories/global-lib.md [hunk 2]
```diff
@@ -15,23 +15,22 @@ npm install popper.js --save
 npm install bootstrap@next --save
```
[the remark this answers is not in the document]
Comment (filipesilva) [reply docs/documentation/stories/global-lib.md]: Fixed

File: docs/documentation/stories/global-lib.md [hunk 3]
```diff
@@ -15,23 +15,22 @@ npm install popper.js --save
...
-  "../node_modules/jquery/dist/jquery.slim.js",
-  "../node_modules/popper.js/dist/umd/popper.js",
-  "../node_modules/bootstrap/dist/js/bootstrap.js"
+  "node_modules/jquery/dist/jquery.slim.js",
+  "node_modules/popper.js/dist/umd/popper.js",
+  "node_modules/bootstrap/dist/js/bootstrap.js"
 ],
 ```
 
 Finally add the Bootstrap CSS to the `apps[0].styles` array:
```
[the remark this answers is not in the document]
Comment (filipesilva) [reply docs/documentation/stories/global-lib.md]: Fixed

**Pull request merged**